aws/resource_aws_lightsail_load_balancer_test.go

```diff
@@ -0,0 +1,305 @@
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "health_check_path"),
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "instance_port"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAWSLightsailLoadBalancer_Tags(t *testing.T) {
+	var conf lightsail.LoadBalancer
+	lightsailLoadBalancerName := fmt.Sprintf("tf-test-lightsail-%d", acctest.RandInt())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheck(t); testAccPreCheckAWSLightsail(t) },
+		IDRefreshName: "aws_lightsail_load_balancer.lightsail_load_balancer_test",
+		Providers:     testAccProviders,
+		CheckDestroy:  testAccCheckAWSLightsailLoadBalancerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSLightsailLoadBalancerConfig_tags1(lightsailLoadBalancerName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAWSLightsailLoadBalancerExists("aws_lightsail_load_balancer.lightsail_load_balancer_test", &conf),
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "health_check_path"),
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "instance_port"),
+					resource.TestCheckResourceAttr("aws_lightsail_load_balancer.lightsail_load_balancer_test", "tags.%", "1"),
+				),
+			},
+			{
+				Config: testAccAWSLightsailLoadBalancerConfig_tags2(lightsailLoadBalancerName),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					testAccCheckAWSLightsailLoadBalancerExists("aws_lightsail_load_balancer.lightsail_load_balancer_test", &conf),
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "health_check_path"),
+					resource.TestCheckResourceAttrSet("aws_lightsail_load_balancer.lightsail_load_balancer_test", "instance_port"),
+					resource.TestCheckResourceAttr("aws_lightsail_load_balancer.lightsail_load_balancer_test", "tags.%", "2"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckAWSLightsailLoadBalancerExists(n string, res *lightsail.LoadBalancer) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		if rs.Primary.ID == "" {
+			return errors.New("No LightsailLoadBalancer ID is set")
+		}
+
+		conn := testAccProvider.Meta().(*AWSClient).lightsailconn
+
+		respLoadBalancer, err := conn.GetLoadBalancer(&lightsail.GetLoadBalancerInput{
+			LoadBalancerName: aws.String(rs.Primary.Attributes["name"]),
+		})
+
+		if err != nil {
+			return err
+		}
+
+		if respLoadBalancer == nil || respLoadBalancer.LoadBalancer == nil {
+			return fmt.Errorf("Load Balancer (%s) not found", rs.Primary.Attributes["name"])
+		}
+		*res = *respLoadBalancer.LoadBalancer
+		return nil
+	}
+}
+
+func TestAccAWSLightsailLoadBalancer_disappear(t *testing.T) {
```

Comment:
see https://github.com/hashicorp/terraform-provider-aws/blob/main/docs/contributing/running-and-writing-acceptance-tests.md#disappears-acceptance-tests

Reply:
Thanks for the feedback, this change will be implemented once the following pull request is merged by @akonrath https://github.com/akonrath/terraform-provider-aws/pull/96